starport/templates/app/stargate/app/simulation_test.go.plush

```diff
@@ -54,14 +54,12 @@ var defaultConsensusParams = &abci.ConsensusParams{
 	},
 }
 
-// go test -benchmem -run=^$ -bench ^BenchmarkSimulation -Commit=true -cpuprofile cpu.out
+// BenchmarkSimulation run the chain simulation
+// Running using starport command:
+// `staport chain simulation -v --numBlocks 200 --blockSize 50`
+// Running as go benchmark test:
+// `go test -benchmem -run=^$ -bench ^BenchmarkSimulation ./app -NumBlocks=200 -BlockSize 50 -Commit=true -Verbose=true -Enabled=true`
 func BenchmarkSimulation(b *testing.B) {
-	simapp.FlagEnabledValue = true
-	simapp.FlagNumBlocksValue = 200
-	simapp.FlagBlockSizeValue = 50
-	simapp.FlagCommitValue = true
-	simapp.FlagVerboseValue = true
```

Comment:
Do you think there can be a way to keep default values and overwrite them with flags?

It's actually practical to run the simulation from an IDE, but it fails if we remove default flags

Reply:
make sense, I will keep the `FlagEnabledValue` and the `FlagCommitValue` hardcoded for the simulation can work and remove it as starport parameter

Reply:
Cool it actually works now with these flags👍